ansible_dynamic_inventory/cli.py

```diff
@@ -5,10 +5,9 @@
 
 
 def _get_version():
+    a = 0
```

Comment:
__[flake8]__
* [F841] local variable 'a' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->